tests/integration/tests/bundles/jammy-yoga.yaml

```diff
@@ -93,9 +93,6 @@ applications:
       openstack-origin: distro
       verbose: true
     constraints: arch=amd64 mem=2048
-  nova-cloud-controller-mysql-router:
```

Comment:
There are still relations for this app.

Reply:
It's duplicated, see line 83. However, it's weird that juju accept this bundle file